manifests/cf-manifest/manifest/020-cf-properties.yml

```diff
@@ -361,6 +361,25 @@ properties:
     links:
       passwd: (( concat "https://login." properties.system_domain "/forgot_password" ))
       signup: (( concat "https://login." properties.system_domain "/create_account" ))
+    oauth:
+      providers:
+        google:
```

Comment:
I think you're saying that it's not that hard to change later and I agree now :)